labs-ios-starter/Model/Apollo/Topic/Topic.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Topic.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+///Leaders can post topics
+struct Topic: Codable {
+    var id: UUID // needs to be same type as identifier/token/etc
+    var joinCode: String // can this be the ID actually?
+    var leaderId: UUID
+    var members: [Member]?
+
+    var topicName: String // is this changeable? if not, change to constant
+    var contextQuestion: Question
+    var requestQuestion: [Question] //if we change this to requestQuestions, we'll need a coding key
+    var responses: [Response]
+
+    var contextId: Int // do we need this?
```

Comment:
Some of these properties aren't on the backend, making local model different

Reply:
Good catch. I'm trying to make sense of it all without API contracts and a live backend... we may end up using some coding keys so our code remains semantic, but that's ok. I'll still merge this in for now so we have something to work with... it'll just require a little refactoring once the backend is live